### ISSUE TITLE
Multinode treepicker can't be used for umbracoInternalRedirectId

### ISSUE THREAD

**Issue by skttl** [Jeavon/Umbraco-Core-Property-Value-Converters#14]
When using a Multinode treepicker for umbracoInternalRedirectId, Umbraco doesn't do the redirect.

**Comment by Jeavon** [comment]
Yeah, why would you use Multinode picker instead of the content picker for that?

**Comment by skttl** [comment]
Because I can have dynamic start nodes in there, and limit which doctypes can be selected.

I made a pull request :) #15
